RELEASE/scripts/autoscend/auto_pre_adv.ash

```diff
@@ -262,6 +262,7 @@ void handlePreAdventure(location place)
 			abort("Tried to be retro but lacking the Continuum Transfunctioner.");
 		}
 		autoEquip($slot[acc3], $item[Continuum Transfunctioner]);
+		addToMaximize("+equip " + $item[Continuum Transfunctioner]);
```

Comment:
When is the maximizer reset? Only potential problem i see is they get stuck in the maximizer until something resets it.